.gitignore_global

```diff
@@ -0,0 +1,37 @@
+ompiled source #
```

Comment:
@a11ydoer, I do not think we want all these changes in the .gitignore for the project; I think they should be in your own global .gitignore.